srcs/SetServer.cpp

```diff
@@ -364,20 +379,16 @@ std::vector<std::string> &Server::getNicknames()
 	return(this->_nicknames);
 }
 
-void Server::setMessage(const char* msg)
+void Server::setMessage(std::string msg)
 {
-	std::string buf;
-
 	std::map<int, Client*>::iterator it;
 	for(it=_clients.begin(); it!=_clients.end(); it++)
 	{
 		int key = it->first;
 		if(key == this->_clientId)
 		{
```

Comment:
setReadbuf takes std::string and that is why it was using assign function. does char * automatically change to std::string?

Reply:
Define variables -> reply.hpp defines (Error replies & normal replies)